sympy/solvers/ode/tests/test_single.py

```diff
@@ -1951,6 +1967,317 @@ def _get_examples_ode_sol_lie_group():
+        'eq': f(x).diff(x, 2) - 4*f(x).diff(x) + 20*f(x),
+        'sol': [Eq(f(x), (C1*sin(4*x) + C2*cos(4*x))*exp(2*x))],
+        'slow': True,
+    },
+
+    'lin_const_coeff_hom_27': {
+        'eq': f(x).diff(x, 4) + 4*f(x).diff(x, 2) + 4*f(x),
+        'sol': [Eq(f(x), (C1 + C2*x)*sin(x*sqrt(2)) + (C3 + C4*x)*cos(x*sqrt(2)))],
+        'slow': True,
+    },
+
+    'lin_const_coeff_hom_28': {
+        'eq': f(x).diff(x, 3) + 8*f(x),
+        'sol': [Eq(f(x), (C1*sin(x*sqrt(3)) + C2*cos(x*sqrt(3)))*exp(x) + C3*exp(-2*x))],
+        'slow': True,
+    },
+
+    'lin_const_coeff_hom_29': {
+        'eq': f(x).diff(x, 4) + 4*f(x).diff(x, 2),
+        'sol': [Eq(f(x), C1 + C2*x + C3*sin(2*x) + C4*cos(2*x))],
+        'slow': True,
+    },
+
+    'lin_const_coeff_hom_30': {
+        'eq': f(x).diff(x, 5) + 2*f(x).diff(x, 3) + f(x).diff(x),
+        'sol': [Eq(f(x), C1 + (C2 + C3*x)*sin(x) + (C4 + C5*x)*cos(x))],
+        'slow': True,
+    },
+
+    'lin_const_coeff_hom_31': {
+        'eq': f(x).diff(x, 4) + f(x).diff(x, 2) + f(x),
+        'sol': [Eq(f(x), (C1*sin(sqrt(3)*x/2) + C2*cos(sqrt(3)*x/2))*exp(-x/2)
+        + (C3*sin(sqrt(3)*x/2) + C4*cos(sqrt(3)*x/2))*exp(x/2))],
+        'slow': True,
+    },
+
+    'lin_const_coeff_hom_32': {
+        'eq': f(x).diff(x, 4) + 4*f(x).diff(x, 2) + f(x),
+        'sol': [Eq(f(x), C1*sin(x*sqrt(-sqrt(3) + 2)) + C2*sin(x*sqrt(sqrt(3) + 2))
+        + C3*cos(x*sqrt(-sqrt(3) + 2)) + C4*cos(x*sqrt(sqrt(3) + 2)))],
+        'slow': True,
+    },
+
+    # One real root, two complex conjugate pairs
+    'lin_const_coeff_hom_33': {
+        'eq': f(x).diff(x, 5) + 11*f(x).diff(x) - 2*f(x),
+        'sol': [Eq(f(x),
+        C5*exp(r1*x) + exp(re(r2)*x) * (C1*sin(im(r2)*x) + C2*cos(im(r2)*x))
+        + exp(re(r4)*x) * (C3*sin(im(r4)*x) + C4*cos(im(r4)*x)))],
+        'checkodesol_XFAIL':True,  #It Hangs
+    },
+
+    # Three real roots, one complex conjugate pair
+    'lin_const_coeff_hom_34': {
+        'eq': f(x).diff(x,5) - 3*f(x).diff(x) + f(x),
+        'sol': [Eq(f(x),
+        C3*exp(r6*x) + C4*exp(r7*x) + C5*exp(r8*x)
+        + exp(re(r9)*x) * (C1*sin(im(r9)*x) + C2*cos(im(r9)*x)))],
+        'checkodesol_XFAIL':True, #It Hangs
+    },
+
+    # Five distinct real roots
+    'lin_const_coeff_hom_35': {
+        'eq': f(x).diff(x,5) - 100*f(x).diff(x,3) + 1000*f(x).diff(x) + f(x),
+        'sol': [Eq(f(x), C1*exp(r11*x) + C2*exp(r12*x) + C3*exp(r13*x) + C4*exp(r14*x) + C5*exp(r15*x))],
+        'checkodesol_XFAIL':True, #It Hangs
+    },
+
+    # Rational root and unsolvable quintic
+    'lin_const_coeff_hom_36': {
+        'eq': f(x).diff(x, 6) - 6*f(x).diff(x, 5) + 5*f(x).diff(x, 4) + 10*f(x).diff(x) - 50 * f(x),
+        'sol': [Eq(f(x),
+        C5*exp(5*x)
+        + C6*exp(x*r17)
+        + exp(re(r18)*x) * (C1*sin(im(r18)*x) + C2*cos(im(r18)*x))
+        + exp(re(r20)*x) * (C3*sin(im(r20)*x) + C4*cos(im(r20)*x)))],
+        'checkodesol_XFAIL':True, #It Hangs
+    },
+
```

Comment:
```The ODE is:
-50*f(x) + 10*Derivative(f(x), x) + 5*Derivative(f(x), (x, 4)) - 6*Derivative(f(x), (x, 5)) + Derivative(f(x), (x, 6))

The expected solution was
[Eq(f(x), C5*exp(5*x) + C6*exp(x*CRootOf(x**5 - x**4 + 10, 1)) + (C1*sin(x*im(CRootOf(x**5 - x**4 + 10, 2))) + C2*cos(x*im(CRootOf(x**5 - x**4 + 10, 2))))*exp(x*re(CRootOf(x**5 - x**4 + 10, 2))) + (C3*sin(x*im(CRootOf(x**5 - x**4 + 10, 4))) + C4*cos(x*im(CRootOf(x**5 - x**4 + 10, 4))))*exp(x*re(CRootOf(x**5 - x**4 + 10, 4))))]

What dsolve returned is:
Eq(f(x), C5*exp(5*x) + C6*exp(x*CRootOf(_x**5 - _x**4 + 10, 0)) + (C1*sin(x*im(CRootOf(_x**5 - _x**4 + 10, 1))) + C2*cos(x*im(CRootOf(_x**5 - _x**4 + 10, 1))))*exp(x*re(CRootOf(_x**5 - _x**4 + 10, 1))) + (C3*sin(x*im(CRootOf(_x**5 - _x**4 + 10, 3))) + C4*cos(x*im(CRootOf(_x**5 - _x**4 + 10, 3))))*exp(x*re(CRootOf(_x**5 - _x**4 + 10, 3))))
```
@oscarbenjamin Here dsolve return solution with dummy vairable and earlier while refactoring I thought that for testing dummy solution would have been in expected solution and wrote the function accordingly.
any suggestions how these cases should be handled?

Reply:
Is the Dummy just the symbol use in CRootOf?

Does `dummy_eq` work in this kind of case?

Reply:
> Is the Dummy just the symbol use in CRootOf?
> 
Yes, I think so.
> Does `dummy_eq` work in this kind of case?

Not sure about this how to handle it. I wonder why this problem was not encountered in master branch because there also same procedure is followed getting dsolve_sol and comparing with expected solution directly.

Reply:
sorry,I misplaced the roots of equation in expected solution, that's why it was failing.